text/0000-additivescenes-and-networkscenemanager-refactoring.md

```diff
@@ -0,0 +1,247 @@
+# NetworkSceneManger Refactoring and Additive Scene Loading
```

Comment:
typo + better name suggestion:
```suggestion
# NetworkSceneManager Robustness and Additive Scene Loading
```

Reply:
I was making some changes already and noticed your comment, so I included that suggestion in the most recent update.